mkdocs.yml

```diff
@@ -27,7 +27,7 @@ nav:
       - "configtool-oidc": commands/configtool-oidc.md
   - "Catalogs":
       - "Overview": catalogs/index.md
-      - "May 18 2023": catalogs/v8-230518-amd64.md
+      - "May 26 2023": catalogs/v8-230526-amd64.md
```

Comment:
Needs to keep the       - "May 18 2023": catalogs/v8-230518-amd64.md I believe

Reply:
I was thinking just listing the "main" ones in the side nav (one per month, and only the last 4 months), the `catalogs/index.md` page has the full list with no ommissions (or should anyway :) )